scripts/starkskills_site/build_site.py

```
@@ -0,0 +1,1145 @@
+
+    try:
+        repo_slug = normalize_repo_slug(args.repo_slug)
+        repo_ref = normalize_repo_ref(args.repo_ref)
+    except ValueError as exc:
+        raise SystemExit(f"[build_site] {exc}") from exc
+    try:
+        domain = normalize_domain(args.domain)
+    except ValueError as exc:
+        if cname_path.exists():
+            cname_path.unlink()
+        raise SystemExit(f"[build_site] {exc}") from exc
```

Ruff (0.15.5)

[warning] Avoid specifying long messages outside the exception class

(TRY003)